packages/scripts/scripts/plugin-zip.js

```diff
@@ -47,10 +50,30 @@ if ( hasPackageProp( 'files' ) ) {
 	);
 }
 
+if ( noRootFolderArg !== undefined ) {
+	zipRootFolder = '';
+	stdout.write( 'Plugin files will be zipped without a root folder.\n\n' );
+} else if ( zipRootFolderArg !== undefined ) {
+	const trimmedZipRootFolderArg =
+		typeof zipRootFolderArg === 'string' ? zipRootFolderArg.trim() : null;
+	if ( trimmedZipRootFolderArg === null ) {
+		stdout.write(
```

Comment:
Nit: this could use some formatting to better indicate that the param was misconfigured and the zip wasn't created.

Reply:
I updated the message to Unable to create zip package: please provide a `--root-folder` name or use `--no-root-folder.